libvmi/driver/kvm/libvirt_wrapper.c

```diff
@@ -68,5 +81,7 @@ status_t create_libvirt_wrapper(kvm_instance_t *kvm)
     wrapper->virDomainResume = dlsym(wrapper->handle, "virDomainResume");
     wrapper->virConnectAuthPtrDefault = dlsym(wrapper->handle, "virConnectAuthPtrDefault");
 
+    wrapper->virDomainQemuMonitorCommand = dlsym(wrapper->handle_qemu, "virDomainQemuMonitorCommand");
+
     return sanity_check(kvm);
```

Comment:
Same here, if the sanity check fails, we need to dlclose both handles.

Reply:
took remark into account